src/encoder.rs

```diff
@@ -2027,7 +2027,8 @@ fn encode_partition_topdown(seq: &Sequence, fi: &FrameInvariants, fs: &mut Frame
         partition = PartitionType::PARTITION_SPLIT;
     } else if bsize > fi.min_partition_size {
         // Blocks of sizes within the supported range are subjected to a partitioning decision
-        rdo_output = rdo_partition_decision(seq, fi, fs, cw, bsize, bo, &rdo_output, pmvs);
+        rdo_output = rdo_partition_decision(seq, fi, fs, cw,
+            w_pre_cdef, w_post_cdef, bsize, bo, &rdo_output, pmvs);
```

Comment:
@xiphmont , I am passing these two pre/post entropy coder state for cdef, pls check whether I do it correctly here and below.